tests/test_helpers/dom_helper.ts

```diff
@@ -131,10 +131,44 @@ export function triggerMouseEvent(
   });
   (ev as any).offsetX = offsetX;
   (ev as any).offsetY = offsetY;
+  dispatchEvent(selector, ev);
+}
+
+export function triggerWheelEvent(
+  selector: string | EventTarget,
+  extra: WheelEventInit = {
+    bubbles: true,
+    deltaMode: WheelEvent.DOM_DELTA_PIXEL, // = 0
+    deltaX: 0,
+    deltaY: 0,
+  }
+) {
+  const ev = new WheelEvent("wheel", { bubbles: true, ...extra });
+  dispatchEvent(selector, ev);
+}
+
+export function triggerKeyDown(selector: string | EventTarget, eventArgs: KeyboardEventInit) {
+  triggerKeyboardEvent(selector, "keydown", { bubbles: true, cancelable: true, ...eventArgs });
```

Comment:
bubble and cancellable are redundant since they are called by triggerKeyboardEvent anyway